assets/js/preventDoubleSubmit/index.js

```diff
@@ -0,0 +1,15 @@
+;(() => {
+  'use strict'
+
+  const disableSubmitButtons = event => {
+    console.log('event', event.currentTarget)
```

Comment:
I'm surprised the linting check didn't say something about this, but looking at the config, `assets/` is on the ignore list.

Reply:
whoops, removed